setup.cfg

```diff
@@ -72,13 +72,12 @@ install_requires =
     protobuf>=4.21.2,<4.22.0
     psutil>=5.6.0
     pyzmq==22.*
-    setuptools>=49
+    setuptools>=49, <67; python_version == "3.7"
```

Comment:
I think the conclusion was that this issue is generic to all Python versions? @MetRonnie